docs/ko/reference/array/unzipWith.md

```diff
@@ -0,0 +1,32 @@
+# unzipWith
+
+입력된 배열을 분리하거나 변환하여 새로운 배열로 반환하는 함수입니다. 이 함수는 중첩된 배열을 받아서 각 요소들을 분리하여 새로운 배열을 만들거나, 선택적으로 제공된 함수를 이용해 변환된 값으로 새로운 배열을 생성합니다.
```

Comment:
```suggestion
결합되어 있는 2차원 배열을 1차원 배열로 풀고, `iteratee` 함수로 값을 변환해서 새로운 배열을 반환해요.
```

Why not make the description a little simpler?

Reply:
That's something I didn't consider carefully. Thank you for the feedback!